src/sentry/api/serializers/models/event.py

```diff
@@ -305,6 +305,10 @@ def serialize(self, obj, attrs, user):
             'eventID': six.text_type(obj.event_id),
             'projectID': six.text_type(obj.project_id),
             'message': obj.message,
+            # XXX(mitsuhiko): This is a temporary legacy code path
+            # remove by merge date of GH-10740 + 90 days
```

Comment:
This is going to require a little reconciliation when I finally get https://github.com/getsentry/sentry/pull/11071 reviewed and merged, but I can deal with that. 

But it does remind me @mitsuhiko you would be an ideal reviewer for that PR :)